src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleAutoTool.kt

```diff
@@ -48,13 +50,27 @@ object ModuleAutoTool : Module("AutoTool", Category.WORLD) {
 
     private val swapPreviousDelay by int("SwapPreviousDelay", 20, 1..100, "ticks")
 
-    private val requreSneaking by boolean("RequireSneaking", false)
+    private val requireSneaking by boolean("RequireSneaking", false)
 
     @Suppress("unused")
     private val handleBlockBreakingProgress = handler<BlockBreakingProgressEvent> { event ->
-        if (requreSneaking && !player.isSneaking) return@handler
+        switchToBreakBlock(event.pos)
+    }
+
+    fun switchToBreakBlock(pos: BlockPos) {
+        if (requireSneaking && !player.isSneaking) {
+            return
+        }
+
+        val blockState = world.getBlockState(pos)
+
+        if (ModuleCivBreak.enabled && ModuleCivBreak.pos == null && blockState.getHardness(
+                mc.world,
+                pos
+            ) > 0F) {
```

Comment:
Detekt probably complained about this line being too long which is a code smell. Long lines aren't a code small because our screens couldn't display a 120 character line. They are bad because there is too much going on in that specific line. This line isn't even complicated or so it is just hard to read because of the many parameters.

My suggestion is to refactor it like so:
```Kotlin
val isCivBreakActive = ModuleCivBreak.enabled && ModuleCivBreak.pos == null
val isDestroyingNotInstant = blockState.getHardness(world, pos) > 0.0F

if (isCivBreakActive && isDestroyingNotInstant) {
    return
}
```
This code is longer vertically, but all you really need to read to know what it does is the last single line. Additionally I think it would make sense if this code was part of `ModuleCivBreak` like `shouldPauseAutoTool(BlockPos)`

Reply:
Yeah this code should be removed as it is a temporary kinda cheap solution. 
``Module Break`` would much rather listen to the user input making this unnecessary.
I'll convert this pr in a draft till I find time to implement this (in about a week).